app/src/main/java/ar/com/wolox/android/example/model/News.kt

```diff
@@ -0,0 +1,29 @@
+package ar.com.wolox.android.example.model
+
+class News(cTitle: String, cText: String, cPicture: String) {
```

Comment:
One important feature of Kotlin is removing getters and setters. Pleasee tell me if you need help with this